x-pack/plugins/ml/public/application/jobs/new_job/pages/new_job/wizard.tsx

```diff
@@ -78,6 +86,43 @@ export const Wizard: FC<Props> = ({
     stringifyConfigs(jobCreator.jobConfig, jobCreator.datafeedConfig)
   );
 
+  useEffect(() => {
+    const estimator = modelMemoryEstimatorProvider(
+      jobCreator.modelMemoryEstimationPayload$,
+      jobValidator.validationResult$
+    );
+
+    const subscription = new Subscription();
+
+    subscription.add(
+      estimator.updates$.pipe(pairwise()).subscribe(([previousEstimation, currentEstimation]) => {
+        // to make sure we don't overwrite a manual input
+        if (
+          jobCreator.modelMemoryLimit === null ||
+          jobCreator.modelMemoryLimit === previousEstimation
+        ) {
+          jobCreator.modelMemoryLimit = currentEstimation;
+          // required in order to trigger changes on the input
+          jobCreatorUpdate();
+        }
+      })
+    );
+
+    subscription.add(
+      estimator.error$.subscribe(error => {
+        notifications.toasts.addError(error, {
+          title: i18n.translate('xpack.ml.newJob.wizard.estimateModelMemoryError', {
```

Comment:
would it be worth showing the error text in this toast? is there useful information passed back?

Reply:
Done in https://github.com/elastic/kibana/pull/60888/commits/47e25ebd0b086c7e17d2c5f5e1233c3dd0aa456e